contracts/KeysManager.sol

```diff
@@ -599,6 +610,12 @@ contract KeysManager is EternalStorage, IKeysManager {
         ] = _success;
     }
 
+    function _setHasMiningKeyBeenRemoved(address _key) internal {
```

Comment:
Please make it `private`. `solhint` says that `Function order is incorrect, internal function can not go after private function`.

Reply:
My fault, I have active `solium` not `solhint`